docs/user/technical-reference/04-30-storage-configuration.md

```diff
@@ -95,3 +95,36 @@ data:
   accessKey: "YWNjZXNzS2V5"
   secretKey: "c2VjcmV0S2V5"
 ```
+
+## GCS
+
+Google cloud storage can be configured using the `spec.storage.gcs` field. The only required field is `bucket` which contains the GCS bucket name. This storage type allows you to provide additional optional configuration, which is described in [DockerRegistry CR](../resources/06-20-docker-registry-cr.md). One of the optional configurations is the `secretName` that contains the authentication method to the GCS storage which is a private service account key in JSON format.
```

Comment:
```suggestion
Google Cloud Storage (GCS) can be configured using the **spec.storage.gcs** field. The only required field is the **bucket**, which contains the GCS bucket name. This storage type allows you to provide additional optional configuration described in [DockerRegistry CR](../resources/06-20-docker-registry-cr.md). One of the optional configurations is the **secretName**, which contains the authentication method to the GCS storage, which is a private service account key in the JSON format.
```

Reply:
If GCS stands for Google Cloud Storage, does it make sense to say `GCS storage`?